### PR TITLE
3.8 strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**
There are some consistency errors in old changelogs, but didn't touch those.
I think the convention is to de-list the string, and fix it then.

### CODE REVIEW

File: OsmAnd/res/values/strings.xml [hunk 1]
```diff
@@ -13,35 +13,35 @@
...
-    <string name="use_login_password">Use login and password</string>
-    <string name="open_street_map_login_mode">You need to login to upload new or modified changes. \n\nYou can log in using the safe OAuth method or use your login and password.</string>
-    <string name="osm_edits_view_descr">You can view all your not yet uploaded edits or OSM bugs in %1$s. Uploaded points don’t show in OsmAnd.</string>
+    <string name="use_login_password">Log in with username and password</string>
+    <string name="open_street_map_login_mode">You need to log in to upload new or modified changes,\n\neither with the safe OAuth method or using your login and password.</string>
+    <string name="osm_edits_view_descr">All your not yet uploaded edits or OSM bugs can be viewed in %1$s. Uploaded points don’t show up in OsmAnd.</string>
     <string name="sign_in_with_open_street_map">Sign in with OpenStreetMap</string>
-    <string name="login_open_street_map_org">Login to OpenStreetMap.org</string>
-    <string name="login_open_street_map">Login to OpenStreetMap</string>
+    <string name="login_open_street_map_org">Login for OpenStreetMap.org</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:41]: Not 100% sure this and the next string isn't "Log in at".(?)

File: OsmAnd/res/values/strings.xml [hunk 2]
```diff
@@ -13,35 +13,35 @@
...
+    <string name="login_open_street_map">Login for OpenStreetMap</string>
     <string name="plugin_global_prefs_info">These plugin setting are global, and apply to all profiles</string>
-    <string name="message_you_need_add_two_points_to_show_graphs">You need to add at least two points</string>
+    <string name="message_you_need_add_two_points_to_show_graphs">Add at least two points</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:44]:
```suggestion
    <string name="message_you_need_add_two_points_to_show_graphs">Add at least two waypoints</string>
```

File: OsmAnd/res/values/strings.xml [hunk 3]
```diff
@@ -51,17 +51,17 @@
...
     <string name="shared_string_graph">Graph</string>
     <string name="use_two_phase_routing">Use 2-phase A* routing algorithm</string>
-    <string name="file_already_imported">File is already imported in OsmAnd</string>
-    <string name="osm_edit_logout_success">Logout successful</string>
+    <string name="file_already_imported">File already imported in OsmAnd</string>
+    <string name="osm_edit_logout_success">Logged out</string>
     <string name="clear_osm_token">Clear OpenStreetMap OAuth token</string>
     <string name="perform_oauth_authorization">Log in via OAuth</string>
-    <string name="perform_oauth_authorization_description">Perform an OAuth Login to use osmedit features</string>
+    <string name="perform_oauth_authorization_description">Perform an OAuth login to use osmedit features</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:64]: Couldn't find out whether "osmedit" is the correct spelling. "OSMedit" or "OSMEdit" seems more likely.

**Comment by sonora** [comment]
Could you please provide separate pull requests for separate  issues. You just cover too many diiferent topics and questions in one PR to be sorted out.

E.g. regarding padding slashes with blanks: That denotes a line break in poetry. Dashes with meaning 'or' are spelled without blanks.

**Comment by comradekingu** [comment]
@sonora I separated out the specific issues I could find already. Will have a look next time.
Don't know what I was thinking with the slashes, usually it is for consistency. In poetry there is only the following space https://www.grammarly.com/blog/slash/ I think.

**Pull request closed without merging**